app/_includes/events.html

```diff
@@ -7,6 +7,14 @@
 			<h1 class="title feature-header">{{site.data[locale].events.title}}</h1>
 		</div>
 	</div>
+	<div class="row" style="text-align: center">
+		<a href="https://docs.google.com/forms/d/e/1FAIpQLScdvsMHLor8mrDrHa-BQs2XzFb96vIrx3u--LHXWd1g7RncPg/viewform">
+			<div class="btn btn-blue arw">{{site.data[locale].events.register}}
+				<img src="{{ site.baseurl }}/assets/graphics/meta/Arrow.svg" alt="learnnow" />
+			</div>
+		</a>
+	</div>
+	<br>
```

Comment:
Having a `<br>` at this place feels a bit non-ideal. Unfortunately this version of bootstrap does not support spacer classes like `mb-2` which would be the ideal solution for this. Maybe add a p-tag and some inline styles?

```
<div class="row text-center">
  <p style="margin-bottom: 20px;">
    <a href="https://docs.google.com/forms/d/e/1FAIpQLScdvsMHLor8mrDrHa-BQs2XzFb96vIrx3u--LHXWd1g7RncPg/viewform" class="btn btn-blue arw">
      Register event
      <img src="/assets/graphics/meta/Arrow.svg" alt="">
    </a>
  </p>
</div>
```